routes/index.js

```diff
@@ -1,4 +1,13 @@
 const express = require('express');
 const router = express.Router();
+const s3Router = require('./s3Router');
+const orderRouter = require('./orderRouter');
+
+
+router.use('/register', s3Router);
+
+
```

Comment:
뷸필요한 공백 삭제해주세요.

구매하기 API에 s3 업로드 코드가 필요한 코드인지 확인해보시고, PR을 분리해주세요.
```suggestion
const orderRouter = require('./orderRouter');

```